docs/docs-source/docs/modules/get-started/pages/how-to-install-and-use-strimzi.adoc

```diff
@@ -0,0 +1,120 @@
+= How to install Strimzi and create a Kafka cluster
+:toc:
+:toc-title: ON THIS PAGE
+:toclevels: 2
+
+include::ROOT:partial$include.adoc[]
+
+This guide shows how to install Strimzi and create a Kafka cluster suitable for use with Cloudflow.
+
+Strimzi is a Kubernetes operator that simplifies the process of running Apache Kafka.
+
+== Prerequisites
+
+=== CLI Tools
+
+Make sure you have the following prerequisites installed before continuing:
+
+- Helm, version 3 or later
+- Kubectl 
+
+== Installing Strimzi
+
+In this guide, we will use Helm to install Strimzi.
+
+Add the Strimzi Helm repository.
+
+  helm repo add strimzi https://strimzi.io/charts/
+
+Update the repository list
+
+  helm repo update
+
+Install the latest version of Strimzi.
+
+  helm install strimzi/strimzi-kafka-operator --namespace default --generate-name --set watchNamespaces="{cloudflow}"
```

Comment:
If you run this, it gets installed in the default namespace. Should this not install in cloudflow namespace? Also, First time I did this it failed, there was no cloudflow namespace yet so I created it first, uninstalled, installed again.

Reply:
Ah I just saw it is specifically set to default namespace...